test/e2e/shared/suite.go

```diff
@@ -145,7 +145,7 @@ func Node1BeforeSuite(e2eCtx *E2EContext) []byte {
 				success = false
 			}
 			return success
-		}, 10*time.Minute, 5*time.Second).Should(BeTrue(), "Should've eventually succeeded creating an AWS CloudFormation stack")
+		}, 30*time.Minute, 15*time.Second).Should(BeTrue(), "Should've eventually succeeded creating an AWS CloudFormation stack")
```

Comment:
Was there a reason you changed this from 10 minutes to 30 mins?

Reply:
Yea, tests where failing on timeouts few times, to reduce re-triggers

Reply:
Seems to still timeout a lot...